src/EnergyPlus/Psychrometrics.hh

```diff
@@ -284,7 +282,7 @@ namespace Psychrometrics {
 
 		// REFERENCES:
 		// see ASHRAE Fundamentals Psychrometric Chapter
-		// USAGE:  hg = PsyHgAirFnWTdb(w,T)
+		// USAGE:  hg = PsyHgAirFnWTdb(T)
```

Comment:
@mbadams5 If w is no longer an argument, then the function name should be changed to PsyHgAirFnTdb.  But my preference would be to leave it as-is for possible future use of w.